### DISCUSSION TITLE
How do I know which units to change

### DISCUSSION THREAD

**Discussion by fervaldies** [idaholab/moose#22786] [Q&A General] [answered]
I am currently working in MOOSE and I am trying to run a model that is in millimiters. The model in the exo file is 200x100x16mm and I would like to know which parameters I have to change to mm as well and which ones not. The parameters that I have are heat transfer coeff,  and emissivity function. The most confusing one is the emissivity function, where I do not know if I should change the units of the htc only or also the boltzmann variable. And also, is the heat transfer coefficient the same in the ADConvectiveHeatFluxBC and ADFunctionRadiativeBC? Could I use both at the same time in the same surface? Thank you in advance.

**Comment by GiudGiud** [comment, marked as answer]
Hello

heat transfer coef is W/m2 right? If so then it needs to be in W/mm2

emissivity is usually unitless so you would not have to change that one. But if it involves the htc then it's likely you ll need to change it.

My advice would be to transform your mesh to be in meters, then you can use the SI unit everywhere.
This is very easy using the TransformGenerator, with a scaling transformation
https://mooseframework.inl.gov/source/meshgenerators/TransformGenerator.html

Guillaume
